routers/api/packages/container/blob.go

```diff
@@ -119,6 +120,17 @@ func getOrCreateUploadVersion(ctx context.Context, pi *packages_service.PackageI
 				log.Error("Error setting package property: %v", err)
 				return err
 			}
+
+			repository, err := repo_model.GetRepositoryByOwnerAndName(
```

Comment:
Yes. 
Let's say "bob" have a git repository called "uno". This repository contains source code for the `alpine`/`debian`/`rpm`/`container`/etc package.
Once "bob" uploads a package (of any type) called "uno" a repository with the source code will automatically be connected to the package since search across repositories is made using package name.